eland/ml/imported_ml_model.py

```diff
@@ -59,14 +63,28 @@ class ImportedMLModel(MLModel):
         - sklearn.tree.DecisionTreeRegressor
         - sklearn.ensemble.RandomForestRegressor
         - sklearn.ensemble.RandomForestClassifier
+        - lightgbm.LGBMRegressor
+            - Categorical fields are expected to already be processed
+            - Only the following objectives are supported
+                - "regression"
+                - "regression_l1"
+                - "huber"
+                - "fair"
+                - "quantile"
+                - "mape"
+            - The following booster types are supported
+                - "gbdt"
+                - "rf"
+                - "dart"
+                - "goss"
         - xgboost.XGBClassifier
-            - only the following operators are supported:
+            - only the following objectives are supported:
                 - "binary:logistic"
                 - "binary:hinge"
                 - "multi:softmax"
                 - "multi:softprob"
         - xgboost.XGBRegressor
-            - only the following operators are supportd:
+            - only the following objectives are supported:
```

Comment:
@tveasey I think the objective can change what values are stored in the leaves. Which might change how inference is done... (storing log likelihood, etc.)

Reply:
It can, but does it for say `pseudohubererror`. I'd be surprised if it did, but maybe it does for some unexpected reason. Did you test this list in the past?

Reply:
@tveasey I did not test it. 

In a separate PR I can add some tests that cycle through objectives to verify that their support (and catch regressions).

Reply:
Ok great. This is really just correcting a typo so agreed no need to worry about it in this PR, just thought I'd ask the question while it occurred to me.